geotrek/tourism/models.py

```diff
@@ -458,6 +458,13 @@ class TouristicEvent(ZoningPropertiesMixin, AddPropertyMixin, PublishableMixin,
         help_text=_("In hours (1.5 = 1 h 30, 24 = 1 day, 48 = 2 days)"),
         validators=[MinValueValidator(0)]
     )
+    price = models.FloatField(
```

Comment:
I advise you to consider a [DecimalField](https://docs.djangoproject.com/en/3.2/ref/models/fields/#decimalfield) here. It will prevent many rounding issues coming with floating numbers. It is not mandatory in your case since there is no computation on that field. But I think it would guarantee we won't have to deal with rounding and/or formatting to display the values properly.

Reply:
Ok, done !

Reply:
I can rebase if necessary